_data/projects/jwtcognito.yml

```
@@ -0,0 +1,11 @@
+name: jwtcognito
+desc: Golang package to ease the use of AWS Cognito JWTs.
+site: https://github.com/bruno-chavez/jwtcognito
+tags:
+- go
+- cognito
+- aws
+- aws-cognito
+upforgrabs:
+  name: good first issue
+  link: https://github.com/bruno-chavez/jwtcognito/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

A simpler version of the same link:

```suggestion
  link: https://github.com/bruno-chavez/jwtcognito/labels/good%20first%20issue
```